resources/helm/dask-gateway/values.schema.yaml

```diff
@@ -478,6 +478,7 @@ properties:
       - loglevel
       - dashboard
       - service
+      - installTraefik
```

Comment:
This is often named `enabled` in Helm charts where a sub-resource is optional, could we rename it to that? What do you think?

Reply:
I am not opposed to doing it that way. The reason I chose to be more specific was because we use ingressroute and middleware in the gateway templates. So it seems a bit wierd to me that we will use and depend on traefik if traefik is set to enabled: false. What do you think?

Reply:
Lets go for a merge, this is a detail of little importance - feel free to open a PR if you think it should be changed, I'll go for a merge!